src/lib389/lib389/cli_idm/client_config.py

```diff
@@ -99,23 +100,27 @@ def sssd_conf(inst, basedn, log, args):
         g_access = groups.get(args.allowed_group)
         ldap_access_filter = 'ldap_access_filter = (memberOf=%s)' % g_access.dn
 
+    ldap_uri_warning = ""
     if inst.ldapuri.startswith("ldapi://"):
-        log.warning("WARNING: ldap_uri starts with ldapi:// - you should review this parameter in the sssd configuration")
+        ldap_uri_warning = "# ⚠️  WARNING: ldap_uri starts with ldapi:// - you should review this parameter"
```

Comment:
Are those special characters there intentionally?

Reply:
Yep, it's a warning character :)